### PR TITLE
post: what's new in ng 4.3

### PR DESCRIPTION
**Pull request by cexbrayat**
Waiting for the official 4.3 release which was planned for today, but will probably be later this month (no RC released yet).

### CODE REVIEW

File: _posts/2017-07-03-what-is-new-angular-4.3.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+title: What's new in Angular 4.3?
+author: cexbrayat
+tags: ["Angular 2", "Angular", "Angular 4"]
+description: "Angular 4.3 is out! Which new features are included?"
+---
+
+Angular 4.3.0 is here!
+
+<p style="text-align: center;">
+  <a href="https://github.com/angular/angular/blob/master/CHANGELOG.md#430-TODO">
```
Comment (clacote) [inline _posts/2017-07-03-what-is-new-angular-4.3.md]: `...-TODO` does not look like a persistent URL

File: _posts/2017-07-03-what-is-new-angular-4.3.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Be warn though: the `/deep/` combinator has been [deprecated from the Shadow DOM spec](https://www.chromestatus.com/features/6750456638341120) and is being removed from all major browsers. So the support will probably also be dropped by Angular in the future. Until then, it is recommended to use `::ng-deep`, and to only use it with emulated view encapsulation.
+
+# Compiler
+
+This is not really a new feature, but I think it is interesting to understand how things work under the hood.
+
+TypeScript 2.3 recently introduced the concept of [transformers](https://github.com/Microsoft/TypeScript/pull/13940) in the compiler API.
+That means some teams can write custom transformations/plugins that are applied to the code compiled by `tsc`. This is not really something that we will write you and me, but Angular has its own compiler called `ngc` that wraps `tsc`.
+
+Until now, the process for `ngc` was to compile your templates to generate TypeScript files, and then call `tsc` to compile your TypeScript code, and the TypeScript files generated to JavaScript. If that sounds strange, you can check out our ebook where we explain all this.
```
Comment (clacote) [inline _posts/2017-07-03-what-is-new-angular-4.3.md]: `and the TypeScript files generated to JavaScript` seems to lack a verb.

Comment (cexbrayat) [inline _posts/2017-07-03-what-is-new-angular-4.3.md]: There's an extra comma that fucks up the meaning, you're right, I'll fix.

File: _posts/2017-07-03-what-is-new-angular-4.3.md (hunk 1 above)
Comment (cexbrayat) [inline _posts/2017-07-03-what-is-new-angular-4.3.md]: It's not, I'm waiting for the stable release to change it (I can't guess the name they'll give it) :)

**Pull request merged**